.eslintrc.js

```diff
@@ -5,7 +5,8 @@ var ERROR = 2;
 module.exports = {
 	env: {
 		'node': true,
-		'es2017': true
+		'es2017': true,
+		'es2020': true
```

Comment:
Not sure if this has any side effects (cc @cb1kenobi)

Reply:
This shouldn't be a problem. Alloy is using the old eslint, but when we upgrade, this config will change and the whole `env` section is different.